ceph/manifest.json

```diff
@@ -12,6 +12,9 @@
   "metric_prefix": "ceph.",
   "metric_to_check": "ceph.write_bytes_sec",
   "name": "ceph",
+  "process_signatures": [
+    "ceph-*"
```

Comment:
Just curious: are we using regex or glob patterns?

Reply:
No, simple matching, I will change it and add ceph-osd, ceph-mon and ceph-mgr 👍